Bowling/NUnitTestBowling/BowlingTests.cs

```diff
@@ -0,0 +1,68 @@
+using NUnit.Framework;
+using System.Runtime.InteropServices;
+using BowlingApp;
+using System.IO;
+using System;
+using System.Collections.Generic;
+
+namespace NUnitTestBowling
+{
+    public class Tests
+    {
+
+        [Test]
+        [TestCase("XXXXXXXXXXXX", "300", Description = "Perfect Game")]
+        [TestCase("9-9-9-9-9-9-9-9-9-9-", "90", Description = "Open Frames with -")]
+        [TestCase("5/5/5/5/5/5/5/5/5/5/5", "150", Description = "All Spares")]
+        [TestCase("548/XX725/7/52X6/5", "158", Description = "Random Open Spares Strikes")]
+        [TestCase("54637281233445625381", "82", Description = "All Open")]
+        public void TestCompleteScoringOfInputString(string marks, string expectedScore)
+        {
+            using (var sw = new StringWriter())
+            {
+                using (var sr = new StringReader(marks))
+                {
+                    Console.SetOut(sw);
+                    Console.SetIn(sr);
+
+                    Frames bowling = new Frames();
+                    //bowling.stepsForGameCalculation();
+
+                    var result = sw.ToString();
+                    Assert.IsTrue(result.Contains(expectedScore));
+                }
+            }
+        }
+
+        [Test]
+        public void TestManualCompleteScoringOfInputString()
+        {
+            using (var sw = new StringWriter())
+            {
+                using (var sr = new StringReader("xxxxxxxxxxxx"))
+                {
+                    Console.SetOut(sw);
+                    Console.SetIn(sr);
+
+                    //Frames bowling = new Frames();
+                    //bowling.stepsForGameCalculation();
+
+                    var result = sw.ToString();
+                    Assert.IsTrue(result.Contains("300"));
+                }
+
+
+            }
+        }
+
+        [Test]
+        public void TestCreateFramesEnsuringACollectionGetsCreated()
+        {
+            Frame frame = new Frame();
+            //List<Frame> t = frame.CreateFrames("XXXXXXXXXXXX");
```

Comment:
This test isn't doing anything?  Are you meaning to implement some more tests still?